rj_gameplay/stp/skill/ball_carrier.py

```diff
@@ -0,0 +1,37 @@
+from abc import ABC, abstractmethod
+from typing import Optional, Type
+
+import numpy as np
+
+import stp.skill as skill
+from stp import role as role
+
+
+class Pass:
+    __slots__ = ["passer", "receiver", "pt"]
+
+    passer: int
+    receiver: int
+    pt: np.ndarray
+
+    def __init__(self, passer: int, receiver: int, pt: np.ndarray):
+        self.passer = passer
+        self.receiver = receiver
+        self.pt = pt
```

Comment:
This should be somewhere else. Also, please specify `pt`'s dimensions.

Also, pass should contain `velocity` and `pass_start_time`.

Reply:
This isn't really meant to be an actual working play, but more as like something to think about how the library should look like in terms of ergonomics and stuff.